package.json

```diff
@@ -47,5 +47,9 @@
         "uuid": "^10.0.0",
         "vite": "^5.4.8"
     },
-    "type": "module"
+    "type": "module",
+    "dependencies": {
+        "@creit.tech/stellar-wallets-kit": "^1.2.3",
+        "lit": "2.8.0"
```

Comment:
not sure why this is here? maybe it was added by accident?

Reply:
Yeah it was, I have removed it ser